src/main/java/graphql/execution/incremental/DeferExecution.java

```diff
@@ -0,0 +1,25 @@
+package graphql.execution.incremental;
+
+import graphql.ExperimentalApi;
+
+import javax.annotation.Nullable;
+
+/**
+ * Represents details about the defer execution that can be associated with a {@link graphql.execution.MergedField}.
+ * <p>
+ * This representation is used during graphql execution. Check {@link graphql.normalized.incremental.NormalizedDeferExecution}
+ * for the normalized representation of @defer.
+ */
+@ExperimentalApi
+public class DeferExecution {
+    private final String label;
+
+    public DeferExecution(String label) {
+        this.label = label;
+    }
+
+    @Nullable
+    public String getLabel() {
+        return label;
+    }
+}
```

Comment:
`DeferExecution` or `DefferredExecution` ??  Naming is hard

Reply:
Yeah. I think both are grammatically and semantically correct, but with slightly different meanings. 

I'll change this one to `DeferredExecution` so it's consistent with `DeferredCall`. Those classes represent things that are more related to action - "an execution that was/is being/will be **deferred**"

I might keep `DeferPayload` as it is. It represents a state - "a payload of type **defer**"